src/Parser/DeathKnight/Blood/Modules/Items/ShacklesofBryndaor.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { formatPercentage } from 'common/format';
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import SPELLS from 'common/SPELLS/index';
+import ITEMS from 'common/ITEMS';
+import Wrapper from 'common/Wrapper';
+import RunicPowerTracker from 'Parser/DeathKnight/Blood/Modules/RunicPower/RunicPowerTracker';
+
+
+class ShacklesofBryndaor extends Analyzer {
+
+  static dependencies = {
+    combatants: Combatants,
+    runicPowerTracker: RunicPowerTracker,
+  };
+
+  on_initialized() {
+    this.active = this.combatants.selected.hasWrists(ITEMS.SHACKLES_OF_BRYNDAOR.id);
+    if (this.combatants.selected.hasTalent(SPELLS.OSSUARY_TALENT.id)) {
+      this.dsCost -= 5;
+    }
+    if (this.combatants.selected.hasBuff(SPELLS.BLOOD_DEATH_KNIGHT_T20_2SET_BONUS_BUFF.id)) {
```

Comment:
Isn't it the 4p that reduces the RP cost?

Reply:
your right. will fix.